### DISCUSSION TITLE
posts/programming/external

### DISCUSSION THREAD

**Discussion by giscus** [movie42/hyunsu#14] [General]
# posts/programming/external

우리 회사는 POS를 중심으로 핵심 비즈니스 로직이 구성되어있다. 나는 프론트엔드 개발자이고
POS나 테이블 오더 같은 기기들의 옵션을 관리하는 어드민과 키오스크, 웨이팅과 같은...

https://www.hyunsu.info/posts/programming/external

**Comment by saengmotmi** [comment]
ㅋㅋㅋ 파이브 라인즈 오브 코드 라는 책을 읽어보시고 거기서 전략 패턴이 언급되는 부분을 살펴보세요
재밌는 것들이 많이 나올겁니다

**Reply by movie42** [reply]
한번 사서 봐야겠군요. 감사합니다. 몇 주전 권한 관련 분기처리가 엄청 많은 이슈가 하나 있었는데 그거 리팩하는데 도움이 많이 되면 좋겠네요! ㅎㅎㅎ
